pages/ja/lb4/express-with-lb4-rest-tutorial.md

```diff
@@ -0,0 +1,327 @@
+---
+lang: ja
+title: 'Loopback REST APIでExpress Applicationを構築する'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: ja_lb4_sidebar
+permalink: /doc/jp/lb4/express-with-lb4-rest-tutorial.html
```

Comment:
Same here

Reply:
Added and modified `layout: translation` and `jp to ja`.